### PR TITLE
fixtures: implemented new fixture loader for plain mysql;

### PR DESCRIPTION
**Pull request by bogdanfinn**

### CODE REVIEW

File: pkg/db/insert_builder.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+//go:generate mockery -name InsertBuilder
+type InsertBuilder interface {
+	Table(table string) InsertBuilder
+	Columns(columns []string) InsertBuilder
+	Values(values ...interface{}) InsertBuilder
+}
+
+type RawInsertBuilder struct {
```
Comment (josketres) [inline pkg/db/insert_builder.go]: Do we need to export this type?
If not I'd be nice to make it private `rawInsertBuilder`

File: pkg/db/insert_builder.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	Builder squirrel.InsertBuilder
```
Comment (josketres) [inline pkg/db/insert_builder.go]: Do we need to export this field?
I'd suggest we make this private (see principle of encapsulation).

File: test/fixtures_mysql_test.go [hunk 3]
```diff
@@ -45,33 +76,48 @@ func mysqlTestFixtures() []*fixtures.FixtureSet {
...
+			Writer:  fixtures.PlainMySqlFixtureWriterFactory(),
+			Fixtures: []interface{}{
+				&fixtures.PlainMySqlFixture{
+					TableName: "mysql_test_models",
+					Rows: []fixtures.PlainMySqlFixtureRow{
+						{
+							"id":   2,
+							"name": "testName2",
+						},
+					},
```
Comment (josketres) [inline test/fixtures_mysql_test.go]: I really like this approach of writing a map!

**Comment by bogdanfinn** [comment]
We updated the squirrel to 1.2.0 due to the fact we need the "Replace Into" while building sql statements.

https://github.com/Masterminds/squirrel/compare/v1.1.0...v1.2.0

File: pkg/fixtures/writer_plain_mysql.go [hunk 4]
```diff
@@ -70,13 +71,12 @@ func (m *plainMySqlFixtureWriter) Write(fs *FixtureSet) error {
 }
 
 func buildSql(fixture *PlainMySqlFixture) (string, []interface{}, error) {
-	insertBuilder := squirrel.InsertBuilder{}.PlaceholderFormat(squirrel.Question)
+	insertBuilder := squirrel.Replace(fixture.TableName).PlaceholderFormat(squirrel.Question)
```
Comment (josketres) [inline pkg/fixtures/writer_plain_mysql.go]: 💅 
```go
insertBuilder := squirrel.Replace(fixture.TableName).
   PlaceholderFormat(squirrel.Question).
   Columns(fixture.Columns...)
```

**Pull request merged**